images/base/files/usr/local/bin/entrypoint

```diff
@@ -88,6 +88,26 @@ fix_product_name() {
   fi
 }
 
+fix_product_uuid() {
+  # The system UUID is usually read from DMI via sysfs, the problem is that
+  # in the kind case this means that all (container) nodes share the same
+  # system/product uuid, as they share the same DMI.
+  # Note: The UUID is read from DMI, this tool is overwriting the sysfs files
+  # which should fix the attached issue, but this workaround does not address
+  # the issue if a tool is reading directly from DMI.
+  # https://github.com/kubernetes-sigs/kind/issues/1027
+  if [[ -f /sys/class/dmi/id/product_uuid ]]; then
+    echo 'INFO: faking /sys/class/dmi/id/product_uuid to be random'
+    cat /proc/sys/kernel/random/uuid > /kind/product_uuid
```

Comment:
@BenTheElder is this compatible with all OSs?

Reply:
i'm still looking into that aspect, it looks like this is standard.